### DISCUSSION TITLE
Import PNG into pedit?

### DISCUSSION THREAD

**Discussion by dbarnett** [replit/kaboom#609] [Ideas]
I'd like to take an existing PNG image and import it into the pedit tool so I can conveniently add more sprite frames to it. Is there any option for something like that?

The only workaround I ended up finding was to inspect the color codes one-by-one and redraw it pixel-by-pixel in pedit.

**Comment by lajbel** [comment]
I think that Pedit is not being updated at the moment, I hope that soon the Replit people will decide to go back to its development.

**Comment by dbarnett** [comment]
Found a better workaround: I dug into the pedit file structure and found the frames are just stored as PNG data URIs, so it's not too hard to take any PNG, convert it to data URI, and paste it into the pedit file before uploading again.

**Reply by lajbel** [reply]
So cool!

**Reply by L3gomancer** [reply]
@dbarnett  I'd be interested to know how you did this in Replit

**Reply by dbarnett** [reply]
It was a while since I did it but IIRC there was a thing to export/import pedit files, I just needed to replace the contents of the pedit files before uploading again (just find the part that says image/png and a bunch of gibberish). Might've also been an img height/width I needed to adjust when the image size was different.
